src/modules/rest/controllers/rest.instance.metagame.controller.ts

```diff
@@ -73,12 +68,6 @@ export class RestInstanceMetagameController {
             @Query('page', OptionalIntPipe) page?: number,
             @Query('pageSize', OptionalIntPipe) pageSize?: number,
     ): Promise<InstanceMetagameTerritoryEntity[]> {
-        const worldObject: {[k: string]: any} = {};
-
-        if (world) {
-            worldObject.world = world;
-        }
-
-        return await this.mongoOperationsService.findMany(InstanceMetagameTerritoryEntity, worldObject, new Pagination({sortBy, order, page, pageSize}));
+        return await this.mongoOperationsService.findMany(InstanceMetagameTerritoryEntity, {world}, new Pagination({sortBy, order, page, pageSize}));
```

Comment:
Presumably this has been applied elsewhere?

Reply:
Yea the mongoOperationService checks for empty objects and deletes keys with undefined values. So we don't need to handle it here

Reply:
Nice, good refactor